### DISCUSSION TITLE
Steve Klabnik - Rust

### DISCUSSION THREAD

**Discussion by hipstersmoothie** [devtools-fm/devtools.fm#146] [Announcements]
## Background

- https://steveklabnik.com/
- https://twitter.com/steveklabnik

## Potential Topics

- Rust
- Rust Core Team
- Rust book
- Community

## Introduction

Andrew: Hello, welcome to the devtools.fm podcast. This is a podcast about developer tools and the people who make them. I'm Andrew and this is my co-host Justin.

Justin: Hey everyone! Our guest today is Steve Klabnik. Steve is a colleague at Oxide Computer Company, formerly a member of the Rust core team, and co-author of the rust book. Steve, it's great to have you! Is there any more you'd like to tell our listeners about yourself?

### Background

- [x] Originally you were very active in the ruby and rails community. What drew your from that to Rust?

### Rust Language

- [x] Why should a developer choose Rust over existing systems programming languages like C or C++?
- [x] What are the defining features of Rust?
- [x] What type of programs are best written in Rust?
- [ ] Do you think that rust will become the defacto language for most things or just for specific things? I.e what do you think about all the rust web frameworks
- [ ] When should a developer _not_ choose Rust?

### Rust Community

- [x] How welcoming is the rust community for beginners?
- [x] What are the best resources for getting started?

### Oxide
 
- [ ] What kind of work do you do at Oxide?

**Comment by hipstersmoothie** [comment]
Tooltips

https://sandpack.codesandbox.io/docs/advanced-usage/nodebox
https://www.clack.cc/

**Comment by just-be-dev** [comment]
Tooltips:
- Get a labeler! (Here's [mine](https://www.amazon.com/Bluetooth-Templates-Smartphone-Organization-Rechargeable/dp/B0B4W92J8V/ref=sr_1_3?crid=2068QRPR1EFHA&keywords=d11%2Blabel%2Bmaker&qid=1676916664&sprefix=d11%2Blabel%2Bmaker%2Caps%2C75&sr=8-3&th=1))
- https://github.com/hyperfiddle/electric

**Comment by steveklabnik** [comment]
https://github.com/dtolnay/cargo-llvm-lines
